src/ashley/editor/decorators.py

```diff
@@ -104,17 +105,20 @@ def inlinetex(props):
     )
 
 
-def render_children(props):
+def ashley_render_children(props):
     """
     Decorator for the blocks in Draft.js ContentState. TEXBLOCK is a custom
     block that is used through atomic blocks.
     """
-    if props.get("block").get("data").get("type") == "TEXBLOCK":
+    if (
+        props.get("block")
+        and props.get("block").get("data")
+        and props.get("block").get("data").get("type") == "TEXBLOCK"
+    ):
```

Comment:
yes but I got paranoid, normally we have to have a type and date otherwise the exporter throw an error but this way is the right way of writing the test if we consider there is a change of not having the total node

Reply:
You could write something shorter like that : 
props.get("block", {}).get("data", {}).get("type") == "TEXBLOCK"